app/controllers/company_controller.rb

```diff
@@ -1,4 +1,4 @@
-class CompaniesController < ApplicationController
+class CompanyController < ApplicationController
```

Comment:
Can you please explain why it was renamed to singular name? It is not rails convention.